providers/goodreads/goodreads.go

```diff
@@ -0,0 +1,25 @@
+package goodreads
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/zquestz/s/providers"
+)
+
+func init() {
+	providers.AddProvider("goodreads", &Provider{})
+}
+
+// Provider merely implements the Provider interface.
+type Provider struct{}
+
+// BuildURI generates a search URL for Goodreads.
+func (p *Provider) BuildURI(q string) string {
+	return fmt.Sprintf("https://www.goodreads.com/search?q=%s", url.QueryEscape(q))
+}
+
+// Tags returns the tags relevant to this provider.
+func (p *Provider) Tags() []string {
+	return []string{"search"}
```

Comment:
I would remove this tag, this isn't a general purpose search provider.

Reply:
@zquestz fixed.